app/src/main/kotlin/org/fossify/calendar/views/SmallMonthView.kt

```diff
@@ -74,6 +83,17 @@ class SmallMonthView(context: Context, attrs: AttributeSet, defStyle: Int) : Vie
 
     override fun onDraw(canvas: Canvas) {
         super.onDraw(canvas)
+
+        fun edgeCaseHorizontalOffset(text: String, textWidth: Int): Float = when (text) {
+            // dates that end with "1" require a horizontal offset for "today" circle to be
+            // visually centered around the text, coefficients were determined by trial and error
+            "1" -> DATE_1_OFFSET_COEFFICIENT * textWidth
+            "11" -> DATE_11_OFFSET_COEFFICIENT * textWidth
+            "21" -> DATE_21_OFFSET_COEFFICIENT * textWidth
+            "31" -> DATE_31_OFFSET_COEFFICIENT * textWidth
+            else -> 0f
+        }
```

Comment:
> //  coefficients were determined by trial and error

I can tell it took a while to tune these, but we don't need them. 

We can set `textAlign` to `Paint.Align.CENTER` at `init` and the math gets simplified:

```kotlin
val fm = paint.fontMetrics
val radius = dayWidth * 0.41f

// simpler logic inside the loop
val centerX = x * dayWidth - dayWidth / 2
val centerY = y * dayWidth - dayWidth / 2
val baselineY = centerY - (fm.ascent + fm.descent) / 2

canvas.drawText(curId.toString(), centerX, baselineY, paint)
if (curId == todaysId && !isPrintVersion) {
    canvas.drawCircle(centerX, centerY, radius, todayCirclePaint)
}
```